src/org-manager/containers/pending-pbas/pending-pbas.component.html

```diff
@@ -33,11 +34,13 @@ <h2 class="govuk-heading-l">
         <a class="govuk-link" [routerLink]="['review', org.organisationId]" routerLinkActive="router-link-active">View</a>
       </td>
     </tr>
-    <tr class="govuk-table__row" *ngIf="orgsWithPendingPBAs.length === 0">
-      <td class="govuk-table__cell" colspan="5">There are no pending PBAs</td>
-    </tr>
   </tbody>
-</table>
+  </table>
+  <ng-template  #noData>
```

Comment:
Same comment as earlier - two spaces before #noData and a redundant space at the end of the span on the next line.

Reply:
I've resolved it as I see you've changed it now.